azurerm/internal/services/netapp/netapp_snapshot_resource.go

```diff
@@ -71,6 +71,8 @@ func resourceNetAppSnapshot() *schema.Resource {
 				ValidateFunc: ValidateNetAppVolumeName,
 			},
 
+			// TODO: remove this in a next breaking changes release since tags are
+			// not supported anymore on Snapshots
```

Comment:
and we should add the 3.0 tag here

```suggestion
			// not supported anymore on Snapshots (todo 3.0)
```